spec/helpers/comments_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the CommentsHelper. For example:
+#
+# describe CommentsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe CommentsHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
Hi @uwadonat, thank you for the review.  I didn't work on the tests because it isn't required for this project but the next project after this. But once I'm done with this project it would be implemented.

Reply:
Ok. make sure also your readme file is updated. so that users may not be redirected to test your project while it is not yet done.